dotnet/ComputerVision/RecognizeText/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.RecognizeText
+{
+    using Newtonsoft.Json.Linq;
+
+    public class Program
+    {
+        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
+        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+
+        static void Main(string[] args)
+        {
+            RecognizeTextSample.RunAsync(endpoint, subscriptionKey).Wait(5000);
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+    }
+
+    public class RecognizeTextSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            Console.WriteLine("Recognizing text from the images:");
+
+            string imageFilePath = @"Images\handwritten_text.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/printed_text.jpg";
+
+            await RecognizeTextFromStreamAsync(imageFilePath, endpoint, key, "Handwritten");  //the last parameter is whether the text that has to be extracted is printed or handwritten 
```

Comment:
Same as the SDK case, we no longer need the mode parameter

Reply:
https://westus.dev.cognitive.microsoft.com/docs/services/5adf991815e1060e6355ad44/operations/587f2c6a154055056008f200
It is required, like in the SDK case